packages/vuetify/src/util/helpers.ts

```diff
@@ -443,3 +443,18 @@ export function getSlotType<T extends boolean = false> (vm: Vue, name: string, s
   if (vm.$slots[name]) return 'normal'
   if (vm.$scopedSlots[name]) return 'scoped'
 }
+
+export function debounce (fn: Function, delay: number) {
+  let timeoutId = 0 as any
+  return (...args: any[]) => {
+    clearTimeout(timeoutId)
+    timeoutId = setTimeout(() => fn(...args), delay)
+  }
+}
+
+export function getPrefixedScopedSlots (prefix: string, scopedSlots: any) {
```

Comment:
This feels very similar to something kael built on the toolbar branch. https://github.com/vuetifyjs/vuetify/pull/6701/files#diff-5f088b65834821e88d7fdbe935ef90c6R435

Reply:
Not really

Reply:
The `genSlots` method in VDataIterator however is pretty much the same function, and can be switched out when things are merged.

Reply:
I said it feels, not that it is! :)